[2024-03-11T21:35:28.066+0530] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: python_operator_dag.taskA scheduled__2024-03-10T00:00:00+00:00 [queued]>
[2024-03-11T21:35:28.076+0530] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: python_operator_dag.taskA scheduled__2024-03-10T00:00:00+00:00 [queued]>
[2024-03-11T21:35:28.076+0530] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-11T21:35:28.104+0530] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): taskA> on 2024-03-10 00:00:00+00:00
[2024-03-11T21:35:28.107+0530] {standard_task_runner.py:60} INFO - Started process 256924 to run task
[2024-03-11T21:35:28.111+0530] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'python_operator_dag', 'taskA', 'scheduled__2024-03-10T00:00:00+00:00', '--job-id', '182', '--raw', '--subdir', 'DAGS_FOLDER/execute_python_operators.py', '--cfg-path', '/tmp/tmpq1ut7z94']
[2024-03-11T21:35:28.112+0530] {standard_task_runner.py:88} INFO - Job 182: Subtask taskA
[2024-03-11T21:35:28.168+0530] {task_command.py:423} INFO - Running <TaskInstance: python_operator_dag.taskA scheduled__2024-03-10T00:00:00+00:00 [running]> on host BreakThrough.
[2024-03-11T21:35:28.363+0530] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='kai' AIRFLOW_CTX_DAG_ID='python_operator_dag' AIRFLOW_CTX_TASK_ID='taskA' AIRFLOW_CTX_EXECUTION_DATE='2024-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-10T00:00:00+00:00'
[2024-03-11T21:35:28.389+0530] {logging_mixin.py:188} INFO - Hello Desmond Rome!
[2024-03-11T21:35:28.389+0530] {python.py:202} INFO - Done. Returned value was: None
[2024-03-11T21:35:28.409+0530] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=python_operator_dag, task_id=taskA, execution_date=20240310T000000, start_date=20240311T160528, end_date=20240311T160528
[2024-03-11T21:35:28.485+0530] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-11T21:35:28.507+0530] {taskinstance.py:3309} INFO - 2 downstream tasks scheduled from follow-on schedule check
